analysed-packages/libseccomp/version-2.5.3/libseccomp-2.5.3-OSS-disclosure.txt

```diff
@@ -0,0 +1,1438 @@
+================================================================================
+
+libseccomp-2.5.3
+
+--------------------------------------------------------------------------------
+
+================================================================================
+
+LICENSES 
+
+--------------------------------------------------------------------------------
+
+warranty-disclaimer
+
+use at your own risk
```

Comment:
I believe that is because of the statement of your lawyer, is this correct? If so it would be nice if you can change "use" to "Use"

Reply:
Done, thank you!